src/java.desktop/share/classes/javax/swing/JTable.java

```diff
@@ -5584,7 +5583,6 @@ public Component getTableCellEditorComponent(JTable table, Object value,
                 if (type == Object.class) {
                     type = String.class;
                 }
-                SwingUtilities2.checkAccess(type.getModifiers());
```

Comment:
```patch
--- a/src/java.desktop/share/classes/javax/swing/JTable.java
+++ b/src/java.desktop/share/classes/javax/swing/JTable.java
@@ -6364,9 +6364,6 @@ public boolean print(PrintMode printMode,
             }
         }
 
-        // Get a PrinterJob.
-        // Do this before anything with side-effects since it may throw a
-        // security exception - in which case we don't want to do anything else.
         final PrinterJob job = PrinterJob.getPrinterJob();
 
         if (isEditing()) {
```